api/scholarqa/scholar_qa.py

```diff
@@ -81,7 +87,7 @@ def update_task_state(
             )
 
     @traceable(name="Preprocessing: Validate and decompose user query")
-    def preprocess_query(self, query: str, cost_args: CostReportingArgs, ) -> CostAwareLLMResult:
+    def preprocess_query(self, query: str, cost_args: CostReportingArgs=None, ) -> CostAwareLLMResult:
```

Comment:
CostReportingArgs can be None when called from the library, the local cost calculation doesn't really need it